docs/api-docs/cart-and-checkout/custom-order-confirmation.md

```diff
@@ -0,0 +1,91 @@
+- [Related resources](#related-resources)
+
+</div>
+
+You can engage with your customers after purchase with a custom order confirmation page. This article outlines how to package a custom order confirmation file, and install a custom confirmation file via the control panel.
+The installation process is the same as [Installing a Custom Checkout](https://developer.bigcommerce.com/stencil-docs/customizing-checkout/installing-custom-checkouts), except you will modify a different [file](https://github.com/bigcommerce/checkout-js/blob/master/src/app/order/OrderConfirmation.tsx) in [Open Source Checkout](https://github.com/bigcommerce/checkout-js).
+
+## Generate the JavaScript loader file
+The first step to installing a custom order confirmation page is generating the loader file. If creating both a custom checkout and a custom order confirmation page using checkout-js, the generated loader file is the same. However, you do not need to create a custom checkout in order to have a custom order confirmation page. You can install a custom order confirmation page and still use the default checkout. The loader file must be responsible for the following:
+
+* Loading all the required assets, including CSS files
+* Running the app
+
+### Packaging custom order confirmation page with Open Source Checkout
+
+If using Open Source Checkout, you can generate the loader file by following these steps:
+
+1. Navigate and locate the Open Source Checkout folder using the terminal, and run `npm ci` to download dependencies
+2. Run `npm run build` to generate a `/dist/` folder with all the files
+
+When using Open Source Checkout to view changes and test implementations through the UI, deploy, and install the custom confirmation page. More details are below.
+
+### Properties available on the custom order confirmation page
+When a store uses a custom confirmation page, some properties are attached to the `window.checkoutConfig` object on the checkout page. These properties are:
+
+* `containerId`: ID of the HTML component where you load the checkout app.
+* `orderId`: if present, it means we should render an order confirmation page for the given order Id
+* `checkoutId`: If present, the ID of the active checkout. You can't have both `orderId` and `checkoutId`
+
+To use the data exposed in the window object, include relevant code in your loader file. Here is an example of this code :
+
+```js
+ReactDOM.render(
+  <OrderConfirmation orderId={ window.checkoutConfig.orderId } />,
+  document.getElementById(window.checkoutConfig.containerId)
+);
+```
+
+## Hosting a custom order confirmation page
+
+You will need to host the custom confirmation file online so it can be served by the store. You can use a hosting service, such as [Amazon S3](https://aws.amazon.com/s3/). Using an external host will allow you the freedom to automate the build process if you wish and push updates automatically from your local machine.
```

Comment:
```suggestion
You will need to host the custom confirmation file online so it can be served by the store using an external hosting service. Using an external host will allow you the freedom to automate the build process if you wish and push updates automatically from your local machine.
```